src/main/java/org/codehaus/mojo/versions/UpdateParentMojo.java

```diff
@@ -134,15 +134,13 @@ protected void update( ModifiedPomXMLEventReader pom )
         ArtifactVersion artifactVersion;
         try
         {
-            artifactVersion = findLatestVersion( artifact, versionRange, false, true,
-                    allowDowngrade );
+            artifactVersion = findLatestVersion( artifact, versionRange, null, false, allowDowngrade );
```

Comment:
this is going back to the original argument list from before the offending PR; that `null` argument means overriding the property accessible to `findLatestVersions` -- I think this is to be refactored, but that's for another time

The other change is setting `usePluginRepositories` to `false` like it was before that PR